src/main/java/com/yahoo/bullet/record/BulletRecord.java

```diff
@@ -112,35 +141,48 @@ public Object get(String field, String subKey) {
      * @param field The field name in the record that is a {@link Map}.
      * @param subKey The subfield map in the {@link Map} that is desired.
      * @param subSubKey The subfield in the subfield {@link Map} that is desired.
-     * @return The value of the subfield in the subfield {@link Map} or null if the field does not exist.
+     * @return The value of the subfield in the map {@link Map} as a {@link TypedObject} or {@link TypedObject#NULL} if the field does not exist.
```

Comment:
subfield since it's a subfield of a subfield..

Reply:
Are you referring to the field in "if the _field_ does not exist"? It returns NULL if anything is missing to the subfield's subfield.